packages/slider/src/presenters/stylesheet.js

```diff
@@ -127,7 +122,7 @@ function stylesheet(props, themeData) {
     },
     {
       mozilla: {
-        ...thumbDisabledRules
+        // ...thumbDisabledRules
```

Comment:
If we no longer need `thumbDisabledRules` here then we should just remove the `mozilla` property from the object.